setup.py

```diff
@@ -19,24 +19,63 @@
 """
 
 import setuptools
+import pathlib
+
+here = pathlib.Path(__file__).parent.resolve()
 
 # Get the long description from the README file
 long_description = (here / 'README.md').read_text(encoding='utf-8')
 
+
 setuptools.setup(
-    name="arc3o-ClimateClara", 
-    version="0.1",
-    author="Clara Burgard",
-    author_email="clara.burgard@gmail.com",
-    description="The Arctic Ocean Observation Operator for 6.9 GHz",
+	
+	#The project's name
+    name='arc3o',
+    
+    #The project's version 
+    version='0.1',
```

Comment:
You should also add this version to the `__init__.py` file (sorry, I forgot to mention it)

see https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names

Reply:
thanks, done!

Reply:
Great! Thanks :smile: 

If you want to be a Github pro, 

- paste the commit SHA here in the comment: edfec6c, it automatically renders to the correct link
- add a note to the comment in your commit message, e.g.
  ```
  added version number in __init__

  see https://github.com/ClimateClara/arc3o/pull/7#discussion_r450296290
  ```

  then it is super-easy to get the context and it renders nicely on github
- click the `Resolve conversation` button below this review comment to indicate that this got resolved (this has been done automatically when you clicked the `Commit suggestion` button for the other comments, but here this was not possible).